ethcore/res/ethereum/morden.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Morden",
-	"dataDir": "test",
+	"dataDir": "Morden",
```

Comment:
Can we change this to lower-case name `morden` to keep it consistent with other chain configs?

Reply:
@sorpaas you are referring to `"dataDir"` right? If, that's the case I totally agree:

```
classic.json:3: "dataDir": "classic",
easthub.json:3:  "dataDir": "easthub",
ellaism.json:3:  "dataDir": "ellaism",
expanse.json:3: "dataDir": "expanse",
foundation.json:3:      "dataDir": "ethereum",
kovan.json:3:   "dataDir": "kovan",
kovan_wasm_test.json:3: "dataDir": "kovan-test",
mcip3_test.json:3:  "dataDir":"mcip3test",
mcip6_byz.json:3:  "dataDir":"mcip6test",
morden.json:3:  "dataDir": "test",
musicoin.json:3:  "dataDir":"musicoin",
ropsten.json:3: "dataDir": "test",
social.json:3:  "dataDir": "social",
```

Reply:
Changed. :)